corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -26,6 +28,11 @@ def get_valid_recipients(recipients):
     """
     from corehq.util.models import BouncedEmail
     bounced_emails = BouncedEmail.get_hard_bounced_emails(recipients)
+    for bounced_email in bounced_emails:
+        email_domain = bounced_email.split('@')[1]
+        metrics_gauge('commcare.bounced_email', 1, tags={
+            'domain': email_domain,
```

Comment:
since we use domain in our metrics pretty consistently to mean cchq project space, we should name this tag something else, like `email_domain`

Reply:
updated